moto/cognitoidp/models.py

```diff
@@ -39,6 +42,10 @@ class UserStatus(str, enum.Enum):
     UNCONFIRMED = "UNCONFIRMED"
     RESET_REQUIRED = "RESET_REQUIRED"
 
+class InvalidPasswordException(Exception):
```

Comment:
Please add this Exception to `exceptions.py`, having all exceptions in one place makes it much more readable. 

Also please check it follows the same standard as the other exceptions in that file: extend JSONRestError, passing a message that explains what's going wrong (`This password was invalid`), etc.

Reply:
Update exceptions.py #5758 
kindly review this.